packages/suite/src/support/messages.ts

```diff
@@ -5462,6 +5462,10 @@ export default defineMessages({
         defaultMessage: 'Destination tag',
         id: 'DESTINATION_TAG',
     },
+    DESTINATION_TAG_SHORT: {
+        defaultMessage: 'DT',
```

Comment:
```suggestion
        defaultMessage: 'Memo/Tag',
```
This would maybe be better?

Reply:
It seems better to me, so I changed it.